### ISSUE TITLE
Keep RxBluetoothKit Alive

### ISSUE THREAD

**Issue by fraune** [Polidea/RxBluetoothKit#420]
It appears that Polidea/RxBluetoothKit has not reviewed or merged any Pull Requests since 2020.

I would like to help keep this library alive. Could someone with the appropriate access help by doing one or more of the following?

* Become active in engaging with the community again
* Grant me permission to review/accept Pull Requests
* Transfer ownership of the library to a willing maintainer

I would love to hear thoughts and ideas around this, since there is not another Bluetooth library for RxSwift of this size and reliability.

_Cross-posted on the RxBluetoothKit Glitter._

_Tagging anyone that GitHub says is on the Polidea team_
@adam-stasiak @agatapst @jakublipinski @kkucharc @literator @potiuk @slawomir-andrian

**Comment by potiuk** [comment]
Poliidea has been acquired and unless some individuals want to keep it running, I think best approach is to fork it and develop on your own. The Apache licence allows it, you can fork it and release/develop on your own. I think there is no need to transfer ownership.

**Comment by fraune** [comment]
My concern is that there are already 300 forks of this repo, and I believe most are for adding a feature or fixing a bug that specifically affects an individual or single team. As time goes on, it will become more difficult to source great code from many individuals across many forks.

**Comment by potiuk** [comment]
I think its up to you to make one that will be the main one. I don't think there is anyone who can transfer ownership to you officially, so you need to make sure to build enouhg of value yourself to make it the "main" fork.

**Comment by potiuk** [comment]
BTW. You can start with gathering those 300 repos and bringing their fixes into your fork and ask them to switch to yours. That's what I'd do if I wanted to provide value for others.

**Comment by fraune** [comment]
I understand. I've never attempted something like that before, but I think it may be worth the effort. Thanks for your perspective and for letting me know that this repo probably won't receive any more updates.

**Comment by jberkhei** [comment]
I'd be interested in helping keep this alive.

**Comment by Serg-Pogrebnyak** [comment]
Some time ago I tried to do something the same, I found info that Polidea was redeemed by another company - Snowflake, and I wrote them and asked to move this repo to open source. I wrote an email to them, tried to make an account on their website, and submitted a technical request, but I had no luck. Maybe if someone contacts them via Linkedin or located in Poland and can call them and tell them about this problem - it would be perfect. But after one and half years no updates about this from my end...

**Comment by uKL** [comment]
The Android counterpart has been moved to the maintainer private github. Maybe there is hope? @dariuszseweryn

**Comment by dariuszseweryn** [comment]
I will check what could be done

**Comment by dariuszseweryn** [comment]
@jberkhei or @Serg-Pogrebnyak is any of you still interested in taking ownership and maintaining this library?

**Comment by Serg-Pogrebnyak** [comment]
@dariuszseweryn I've made a fork and supports my version, but if we can move it to the open source - that will be great! Do you have any news about that?

**Comment by dariuszseweryn** [comment]
Yeah, it took a while but I got the necessary approvals. If you would like to take the ownership of the repo going forward it is possible to transfer it to your GitHub account. (That being said I am unsure how this will behave if you have a Fork already but maybe it would just merge?)

I would need your legal name and email address so appropriate documents could be sent.

**Comment by jberkhei** [comment]
I'd be happy to help as well, I am using this library via my fork on numerous projects.  Not sure I want\need ownership.

**Comment by Serg-Pogrebnyak** [comment]
@dariuszseweryn Thank you so much for your trust, but I would prefer for this library to be transferred to the community and maintained collectively.

Mr. @DaBs previously [mentioned](https://github.com/Polidea/RxBluetoothKit/issues/398#issuecomment-915204232) a proposal about moving this repository to the community organization. Mr. @DaBs could you help with this process?

@dariuszseweryn I truly appreciate all the work you’ve put into this and the fact that this issue hasn’t been forgotten. Thanks again!

**Comment by DaBs** [comment]
@Serg-Pogrebnyak At the time that I mentioned it, the [abandonware](https://github.com/abandonware) organisation seemed to be the best place, but I'm unsure if it still is? The abandonware process is relatively straight forward, and [documented here](https://abandonware.github.io/). That being said, I'm not in a capacity where I'm actively using this project myself, so open-source maintaining it would probably just lead to the same stagnation.
